examples/chip-tool/commands/clusters/ReportCommand.h

```diff
@@ -106,32 +108,95 @@ class ReportCommand : public ModelCommand, public chip::app::ReadClient::Callbac
     void OnSubscriptionEstablished(uint64_t subscriptionId) override { OnAttributeSubscription(); }
 
 protected:
-    CHIP_ERROR ReportAttribute(ChipDevice * device, chip::EndpointId endpointId, chip::ClusterId clusterId,
-                               chip::AttributeId attributeId, chip::app::ReadClient::InteractionType interactionType,
-                               uint16_t minInterval = 0, uint16_t maxInterval = 0,
+    CHIP_ERROR ReportAttribute(ChipDevice * device, std::vector<chip::EndpointId> endpointIds,
+                               std::vector<chip::ClusterId> clusterIds, std::vector<chip::AttributeId> attributeIds,
```

Comment:
@vivien-apple when you add vector support for multiple attribute in chip-tool,  could you help extend vector support for the below optional DataVersion, DataVersion filter is per path.

@mrjerryjohns @bzbarsky-apple